max-wave.lua

```diff
@@ -0,0 +1,59 @@
+--Dynamically limit the next immigration wave
+--By Loci, modified by Fleeting Frames and Tachytaenius
+--[====[
+
+max-wave
+========
+Limit the number of migrants that can arrive in the next wave by overriding the population cap value in data/init/d_init.txt (not safe with gui/settings-manager)
```

Comment:
Perhaps explain (and correct me if I'm wrong) that the danger is that gui/settings-manager will write this "temporary" population cap to the d_init.txt file and make it permanent. But that will only happen if the user changes the pop cap in the gui/settings-manager UI, right?

Reply:
Yes.